src/Grpc.Net.Client/Balancer/IChannelControlHelper.cs

```diff
@@ -0,0 +1,52 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+namespace Grpc.Net.Client.Balancer
+{
+    /// <summary>
+    /// Provides essentials for <see cref="LoadBalancer"/> implementations.
+    /// </summary>
+#if HAVE_LOAD_BALANCING
+    public
+#else
+    internal
+#endif
+        interface IChannelControlHelper
```

Comment:
I'm not a fan with a Helper in the name of a public interface. Maybe LoadBalancerController since it seems to interact and "control" LB behaviour?

Reply:
I'm not a fan of the name either but it is consistent with other gRPC load balancer implementations. They all use this name.